src/SrBot.ts

```diff
@@ -87,17 +93,11 @@ export class SrBot {
     }
 
     /**
-     * Returns a StringBuilder of the player's SR in descending order
-     * @param {Player[]} players List of players to build text from
-     * @param {Discord.RichEmbed} embed Existing embed to append to
+     * Sorts a list of players by SR descending
+     * @param {Players[]} players Players to sort
      */
-    private appendPlayersToEmbed(players: Player[], embed: Discord.RichEmbed) {
-        players.sort((a: Player, b: Player) => (a.SR < b.SR) ? 1 : -1)
-            .forEach((player) => {
-                embed = embed.addField(player.player, `${player.SR}${player.private ? ' [PRIVATE]' : ''} ${getRankEmoji(player.SR)}`);
-            });
-
-        return embed;
+    private sortPlayersBySR(players: Player[]) {
+        return players.sort((a: Player, b: Player) => (a.SR < b.SR) ? 1 : -1);
```

Comment:
So I was thinking further on this - Would we just want to sort the players by SR when we write to the file?  Then they'll always be in order.  We can leave this for another PR though